internal/fsutil/create_test.go

```diff
@@ -209,15 +284,26 @@ func (s *S) TestCreate(c *C) {
 
 		c.Assert(err, IsNil)
 		c.Assert(testutil.TreeDump(dir), DeepEquals, test.result)
+
 		// [fsutil.Create] does not return information about parent directories
 		// created implicitly. We only check for the requested path.
-		entry.Path = strings.TrimPrefix(entry.Path, dir)
-		// Add the slashes that TreeDump adds to the path.
-		slashPath := "/" + test.options.Path
-		if test.options.Mode.IsDir() {
-			slashPath = slashPath + "/"
+		if entry.HardLink {
+			// We should test hard link entries differently to ensure that it
+			// produces a hard link indeed.
```

Comment:
```suggestion
			// We should test hard link entries differently because fsutil.Create
			// does not return hash or size when it creates hard links.
```